routes/jpeg/jpeg.js

```diff
@@ -0,0 +1,50 @@
+const retryAfter = require('../retryAfter')
+const { Writable } = require('stream')
+
+const image = (req, res) => {
+  const { pipe2jpeg: { jpeg } = {} } = res.locals
```

Comment:
This probably looks weird, destructuring with a default. I wasn't sure if `pipe2jpeg` would always be set in `res.locals` and added a default to avoid the app needlessly crashing.